### DISCUSSION TITLE
Wiki issue: PIA Port Forwarding warning note

### DISCUSSION THREAD

**Discussion by someoneelsehere** [qdm12/gluetun#648] [Q&A]
Link to problematic page in Wiki: https://github.com/qdm12/gluetun/wiki/Private-internet-access#vpn-server-port-forwarding

What's the issue?
Under https://github.com/qdm12/gluetun/wiki/Private-internet-access#vpn-server-port-forwarding you have noted an issue seemingly on PIA's side. Could it be related to their new next-gen port forwarding API?
https://www.privateinternetaccess.com/helpdesk/kb/articles/next-generation-port-forwarding

**Comment by qdm12** [comment]
Hi there! Thanks for taking the time to report this.

Actually no, it does use PIA's nextgen port forwarding already.

But running a simple test `ncat` or http server listening on the forwarded port cannot be accessed through `<vpn-public-ip>:<forwarded-port>`. It does however work with a few applications such as torrenting, pointing that maybe they block certain type of traffic.

It could also be a bug on their side, maybe it's fixed today, feel free to try it out. You can follow https://github.com/qdm12/gluetun/wiki/VPN-server-port-forwarding#test-it to test it with an http server. I tested this method with other VPN providers supporting port forwarding and it does work.
